WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -558,11 +558,14 @@
           regionalSingleRecord: "NAR",
           regionalAverageRecord: "",
         )
+        expect(competition.results_posted_at).to be nil
         get :post_results, id: competition
```

Comment:
(Not related to your PR, but: According to the HTTP standard `GET` requests should be side-effect free.)